db/schema.rb

```diff
@@ -11,14 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-
 ActiveRecord::Schema.define(version: 20161115154942) do
-  
-  
+
```

Comment:
It is best to not make frivolous stylistic changes such as this. An example of why can be seen in the merge conflict for schema, which would not exist had this file not been edited.